### PR TITLE
Wiggle sort in C

### LINKED ISSUE #5197: Wiggle sort in C
## 🚀 Feature

To reorder an unsorted array such that arr[0]<arr[1]>arr[2]<arr[3]...

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by aish2002**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added wiggle sort in C

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Fixes: #5197

### CODE REVIEW

**Comment by aish2002** [comment]
@ankitaggarwal23 , @HarshCasper , @siddharth25pandey  kindly review the PR

File: C/sort/Wiggle_sort.c [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    //to wiggle sort arr
+    wiggle_sort(no_of_elements, arr);
+    return 0;
+}
+
+/*1. Sample input:
+     6
+     1 3 2 2 3 1
+     Sample output:
+     2 3 1 3 1 2
```
Comment (ankitaggarwal23) [inline C/sort/Wiggle_sort.c]: Use proper display message how you are reading and writing outputs

File: C/sort/Wiggle_sort.c [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+  2. Sample input:
+     10
+     1 4 6 2 3 7 9 2 1 0
+     Sample Output:
+     2 9 2 7 1 6 1 4 0 3
+*/
+
+/*
```
Comment (ankitaggarwal23) [inline C/sort/Wiggle_sort.c]: remove extra test case

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by aish2002** [comment]
Necessary changes are made. @ankitaggarwal23 kindly re-review the PR

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**